cxmon/src/mon.cpp

```diff
@@ -1030,6 +1031,63 @@ void mon_change_dir()
 
 
 /*
+ * Add break point
+ */
+
+void mon_add_break_point(uintptr adr)
+{
+	BREAK_POINT_SET::iterator it;
+	// Save break point
+	if ((it = disabled_break_points.find(adr)) == disabled_break_points.end())
+		active_break_points.insert(adr);
+	else {
```

Comment:
Nit: Use {}'s in both sides of the if/else

Reply:
done